prov/shm/src/smr_ep.c

```diff
@@ -391,6 +391,20 @@ int smr_format_ze_ipc(struct smr_ep *ep, int64_t id, struct smr_cmd *cmd,
 	return FI_SUCCESS;
 }
 
+int smr_format_ipc(struct smr_cmd *cmd, const struct iovec *iov,
+                   size_t total_len, struct smr_region *smr,
+                   struct smr_resp *resp, enum fi_hmem_iface iface)
+{
+    cmd->msg.hdr.op_src = smr_src_ipc;
+    cmd->msg.hdr.src_data = smr_get_offset(smr, resp);
+    cmd->msg.hdr.size = total_len;
+    cmd->msg.data.ipc_info.iface = iface;
+
+    return ofi_hmem_get_handle(cmd->msg.data.ipc_info.iface,
+                               iov[0].iov_base,
```

Comment:
I wouldn't take an iovec as a param to this function if it doesn't support more than one.